### ISSUE TITLE
Can't login with post '/user/login/', { :email => 'username', :password => 'password' }

### ISSUE THREAD

**Issue by dmotrenko** [brooklynDev/airborne#73]
I am trying to login 
I tried 
post '/user/login/', { :email => 'username', :password => 'password' }
Tried
post '/user/login/', { :email => 'username', :password => 'password' } , { 'Content-Type' => 'x-www-form-urlencoded' }
and get an error message

I can login with postman using the same credentials
I can login   with  frisby.js 
frisby.create('login')
  .post(BASE_URL + 'user/login',
    {
        email: email,
        password: password
    }
    )
  .expectStatus(200)
  .expectJSON({
    status: 0
})

**Comment by sethpollack** [comment]
Is Airborne working for you at all? Are you using rails?

**Comment by dmotrenko** [comment]
Yes, airborne is working for get requests. And I don't use rails.

**Comment by brooklynDev** [comment]
I tried reproducing this with no luck. Can you post the error message you're getting?

**Comment by dmotrenko** [comment]
describe 'catalog paid videos' do
  it 'should validate types' do
    post '/user/login', { :email => 'tvzavr@list.ru', :password => 'qazwsx' }, { 'Content-Type' => 'application/x-www-form-urlencoded' }
    puts response
	end
end

this is the response
{"status":61,"error":[{"description_ru":"поле пустое"},{" description_en":"Field empty"}]}
This is custome error message means that I am not logged in. 

This is the code I use in frisby.js
frisby.create('login')
  .post(BASE_URL + 'user/login',
    {email: email, password: password}
    )
  .expectStatus(200)
  .expectJSON({
    status: 0
})
And this is works all the time. I get  status: 0 as expected and user is logged in.

**Comment by aditya-freshdesk** [comment]
Is the endpoint right? Or missing base URL

**Comment by docljn** [comment]
Hello.  Code Triage volunteer here.  I've just started using airborne and am really enjoying it.

As this issue has been open for several years, and could not be replicated, I would like to propose that it be closed @dmotrenko @brooklynDev
